listenbrainz/webserver/static/js/src/utils.tsx

```diff
@@ -54,6 +54,58 @@ const searchForSpotifyTrack = async (
   return null;
 };
 
+const searchForYoutubeTrack = async (
+  apiKey?: string,
+  accessToken?: string,
+  trackName?: string,
+  artistName?: string,
+  releaseName?: string,
+  refreshToken?: () => Promise<string>
+): Promise<Array<string> | null> => {
+  if (!apiKey) return null;
+  if (!accessToken) return null;
+  let query = trackName;
+  if (artistName) {
+    query += ` ${artistName}`;
+  }
+  if (releaseName) {
+    query += ` ${releaseName}`;
+  }
+  const response = await fetch(
+    `https://youtube.googleapis.com/youtube/v3/search?part=snippet&q=${query}&key=${apiKey}`,
```

Comment:
```suggestion
    `https://youtube.googleapis.com/youtube/v3/search?part=snippet&type=video&q=${query}&key=${apiKey}`,
```

According to https://developers.google.com/youtube/v3/docs/search/list#type

We could maybe also want to set the `videoEmbeddable` parameter to true, since we're only looking to embed the result videos